Obsidian/Daratrine/01_Worldbuilding/01.3_Geografia_e_Arquitetura/01.3.1_Ekios/1. Índice Geral Geografia Arquitetura.md

```
@@ -14,14 +14,14 @@ Esse docuemnto [[Índice_Geral_Geografia_Arquitetura]] funciona como uma camada
-Continente de Ekios: [[Geografia_e_arquitetura_continente_Ekios]]
-Gildrat: [[01.3.2_Gildrat]]
+Continente de Ekios: [[Continente_Ekios_Geral]]
+Gildrat: [[Gildrat]]
 Locais em Gildrat: *Daratrine/worldbuilding/01.3_Geografia_e_Arquitetura/0.3.1_Gildrat/01.3.2_Locais_*
   
 ## Como Usar
 
-  1. Usar  [[Geografia_e_arquitetura_continente_Ekios]] para decisões macro: biomas, rotas, corpos d'agua, clima, astronomia e zonas de risco.
-2. Usar [[01.3.2_Gildrat]] para decisões de arquitetura anã, distritos, infraestrutura e identidade urbana.
+  1. Usar  [[Continente_Ekios_Geral]] para decisões macro: biomas, rotas, corpos d'agua, clima, astronomia e zonas de risco.
+2. Usar [[Gildrat]] para decisões de arquitetura anã, distritos, infraestrutura e identidade urbana.
```

markdownlint-cli2 (0.23.0)

[warning] Inconsistent indentation for list items at the same level
Expected: 2; Actual: 0

(MD005, list-indent)